### PR TITLE
feat:Added functionality to delete multiple faq

### LINKED ISSUE #921: Selecting multiple FAQ's at once
### Feature Request

**Feature Description**
Currently in the FAQ list mode there is an option to delete the faq's but only 1 faq can be selected and deleted at a time. I would like to add a feature where multiple FAQ's can be selected and deleted at once.

**Would you like to work on the issue?**
Yes!

@iamareebjamal Can I work on this issue?

### PR DESCRIPTION
**Pull request by sridharjajoo**
Fixes #921 

Changes:
- The code for creating the delete option on the toolbar has been removed because, the ActionMode  handles it.
- In the ActionMode, the color of the status bar gets changed to black and hence it needs to be changed to color_top_surface, this is handled in the onCreateActionsMode( ).[[link](https://stackoverflow.com/questions/29618470/change-status-bar-color-when-entering-contextual-action-mode)]

Screenshots for the change: 
ezgif com-resize 9

### CODE REVIEW

**Comment by Masquerade0097** [comment]
@sridharjajoo We are moving towards Contextutal Action Bar #1042 for implementing deleting and updating features. The currently used way is non standard.

**Comment by sridharjajoo** [comment]
@Masquerade0097 @iamareebjamal @mishuvs I have included the contextual action bar in the code. Kindly review

**Comment by iamareebjamal** [comment]
@Masquerade0097 @mishuvs @srv-twry Please review

File: app/src/main/java/org/fossasia/openevent/app/core/faq/list/FaqListFragment.java [hunk 1]
```diff
@@ -49,8 +50,9 @@
     private FaqListAdapter faqsAdapter;
     private FaqsFragmentBinding binding;
     private SwipeRefreshLayout refreshLayout;
-
+    private ActionMode mActionMode;
```
Comment (srv-twry) [inline app/src/main/java/org/fossasia/openevent/app/core/faq/list/FaqListFragment.java]: We don't follow Hungarian notation in this project

**Comment by sridharjajoo** [comment]
@srv-twry I have done the required change. Kindly re-review

**Comment by Masquerade0097** [comment]
@sridharjajoo Please post a gif for the changes done so that everyone can see the behavior here only.

**Comment by sridharjajoo** [comment]
@Masquerade0097 I have updated the gif above. Kindly check

**Pull request merged**